Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -245,14 +269,40 @@ public class ArcGISARView: UIView {
             subview.bottomAnchor.constraint(equalTo: self.bottomAnchor)
             ])
     }
+    
+    /// Internal method to perform a hit test operation to get the transformation matrix representing the corresponding real-world point for `screenPoint`.
+    ///
+    /// - Parameter screenPoint: screenPoint: The screen point to determine the real world transformation matrix from.
+    /// - Returns: An `AGSTransformationMatrix` representing the real-world point corresponding to `screenPoint`.
+    fileprivate func internalHitTest(screenPoint: CGPoint) -> AGSTransformationMatrix? {
+        // Use the `hitTest` method on ARSCNView to get the location of `screenPoint`.
+        let results = arSCNView.hitTest(screenPoint, types: [.existingPlane, .estimatedHorizontalPlane])
+        
+        // Get the worldTransform from the first result; if there's no worldTransform, return nil.
+        guard let worldTransform = results.first?.worldTransform else { return nil }
+        
+        // Create our hit test matrix based on the worldTransform location.
+        let hitTestMatrix = AGSTransformationMatrix(quaternionX: 0.0,
+                                                    quaternionY: 0.0,
+                                                    quaternionZ: 0.0,
+                                                    quaternionW: 1.0,
+                                                    translationX: Double(worldTransform.columns.3.x),
+                                                    translationY: Double(-worldTransform.columns.3.z),
```

Comment:
Just wanted to check that z to Y and y to Z is on purpose.

Reply:
Yes, that is on purpose.